cedar-policy/src/tests.rs

```diff
@@ -3443,3 +3443,37 @@ mod issue_619 {
         );
     }
 }
+
+mod macro_tests {
+    use std::str::FromStr;
+
+    use crate::euid;
+
+    use crate::EntityId;
+    use crate::EntityUid;
+
+    #[test]
+    fn pass() {
+        let e: EntityUid = euid!(Foo, "bar");
```

Comment:
Would it be possible let the macro be written like `euid!(Foo::"bar")`? The way you have it looks fine, but IMO it would be even nice to use the exact euid syntax.

Reply:
I prefer the existing one because it explicitly requires users to adopt Rust strings as `eid`s, thus avoiding to reinterpret the `eid` component.

Reply:
That being said, I'm still open to suggestions here.